physionet-django/console/templates/console/console_navbar.html

```diff
@@ -82,6 +82,9 @@
           <li class="nav-item {% if past_credentials_nav %}active{% endif %} ">
             <a id="nav_past_credential_applications" href="{% url 'past_credential_applications' 'successful' %}">Past Applications</a>
           </li>
+          <li class="nav-item {% if known_ref_nav %}active{% endif %} ">
+            <a id="nav_known_ref" href="{% url 'known_referencess' %}">Known references</a>
```

Comment:
Nitpick but the other items in this particular list are in title case, so it should perhaps be `Known References` for consistency. We are generally pretty inconsistent about casing and there is a mix in the menu.

Reply:
Fixed in next commit :)